src/main/java/com/newkodilla/patterns2/decorator/pizza/PizzaOrder.java

```diff
@@ -0,0 +1,7 @@
+package com.newkodilla.patterns2.decorator.pizza;
+
+public interface PizzaOrder
+{
+    public double getPrice();
+    public String getDescription();
```

Comment:
pisanie 'public' w interfejsie jest redundantne i można spokojnie skasować

Reply:
Tak, ale konsekwentnie piszę public, bo tak przyjąłem, jak również this. w getterach i nie stosuję importu statycznego.